demo/.eslintrc

```diff
@@ -12,6 +12,8 @@
     "class-methods-use-this": 0,
     "no-plusplus": 0,
     "semi": ["error", "never"],
-    "react/require-default-props": "off"
+    "react/require-default-props": "off",
+    "react/jsx-max-props-per-line" : ['error', {maximum: 1}]
```

Comment:
Not sure I agree with this.  Sometimes we want props on a single line depending on the line length.